samples/hello-oboe/CMakeLists.txt

```diff
@@ -53,3 +53,7 @@ add_library(hello-oboe SHARED
 
 # Specify the libraries needed for hello-oboe
 target_link_libraries(hello-oboe android log oboe)
+
+# Enable optimization flags: if having problems with source level debugging,
+# disable -Ofast ( and debug ), re-enable after done debugging.
+target_compile_options(hello-oboe PRIVATE -std=c++14 -Wall -Werror -Ofast)
```

Comment:
I think the only reason for using C++14 is that the sample is using `std::make_unique` which I believe it isn't.

Reply:
done